docs/notebooks/documents/getting_started.py

```diff
@@ -0,0 +1,136 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.16.1
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Getting Started
+#
+# This document provides an overview for preparing to work with PDO contracts through the contract
+# docker container. It will walk through the necessary steps to set up a functional configuration.
+#
+# %% [markdown]
+# ## How Contracts Use Jupyter
+#
+# Many of the PDO contract families provide Jupyter notebooks to simplify interaction with contract
+# objects. Commonly, the notebooks provide a factory for instantiating a notebook for each new
+# contract from one of the contract specific templates.The templates provided generally share five
+# common sections:
+#
+# * Configure the Contract Object
+# * Initialize the Interpreter and the PDO Environment
+# * Initialize the Contract Object Context
+# * Operate on the Contract
+# * View Contract Metadata
+#
+# To initialize the interpreter, the notebook loads the Juptyer helper module. This module imports
+# all of the relevant PDO and Jupyter modules to simplify access for code in the notebook. It also
+# defines several procedures that are useful for initializing and interacting with the environment.
+#
+# In addition, the interpreter initialization configures an IPython extension that makes it easier
+# to provide code for multiple types of operations that can be performed on the contract
+# object. Specifically, the extension defines a magic keyword, `skip`, that takes a value or
+# expression that, if it evaluates to True, causes the code section to be skipped during notebook
+# execution.
+#
+# The next section in the notebook configures information about the contract object associated with
+# the notebook. In some cases, the variables will set by the notebook using the Papermill
```

Comment:
"The next section in the notebook " and the rest of the of the documentation in this section. sounds like what "this specific notebook does" whereas I guess you are referring to notebook that is used to interact with contracts?

Reply:
suggest wording changes.